tests/integration/models/device.spec.js

```diff
@@ -2261,6 +2261,12 @@ describe('Device Model', function () {
 					// Augmented properties
 					// Should filter out deleted image installs
 					expect(deviceDetails.current_services.db).to.have.lengthOf(1);
+					const currentServices =
+						_.flatten(Object.values(deviceDetails.current_services)) ?? [];
+					currentServices.forEach((currentService) => {
+						expect(currentService).to.have.property('commit');
+						expect(currentService).to.have.property('release_id');
```

Comment:
You can actually also add this a few lines above after the `expect(imageInstall).to.not.have.property('commit');` since the `current_services` is just a dictionary of the image installs :)

Reply:
Do you mean right after the `forEach` ends? Or *inside* the forEach before it ends?

Reply:
I mean on line 2256
https://github.com/balena-io/balena-sdk/pull/1204/files#diff-64d3e0fa4d8bb01064ac099bacb99a8bff3c732683a5931c7edd36e7a401bb8eR2256
![image](https://user-images.githubusercontent.com/1295829/157871255-68cc30d9-1e9a-4e07-91d5-ead96f026a01.png)

Reply:
Yes but I do not understand why we can put these lines right after that line. If I put them there as is, then that check will run for each imageInstall. If you meant I should put the checks there alter them a bit, what kind of alteration would have been necessary? Because `imageInstall` does not immediately have the `release_id` property on it so I can't just do `expect(imageInstall).to.have.property('release_id')`
I'm very likely missing something 😅

Reply:
Nevermind that's already fine anyway :+1: